Library/EnvelopeFactory.ts

```diff
@@ -356,7 +356,7 @@ class _StackFrame {
 
     // regex to match stack frames from ie/chrome/ff
     // methodName=$2, fileName=$4, lineNo=$5, column=$6
-    public static regex = /^([\s]+at)?(.*?)(\@|\s\(|\s)([^\(\@\n]+):([0-9]+):([0-9]+)(\)?)$/;
+    public static regex = /^([\s]+at)?(.*?)(\@|\s\(|\s)([^\(\n]+):([0-9]+):([0-9]+)(\)?)$/;
```

Comment:
This can be simplified:
```suggestion
    public static regex = /^(\s+at)?(.*?)(\@|\s\(|\s)([^\(\n]+):(\d+):(\d+)(\)?)$/;
```